seaborn/categorical.py

```diff
@@ -1230,24 +1230,49 @@ def estimate_statistic(self, estimator, ci, n_boot):
             self.value_label = "{}({})".format(estimator.__name__,
                                                self.value_label)
 
-    def draw_confints(self, ax, at_group, confint, colors, **kws):
+    def draw_confints(self,
+                      ax,
+                      at_group,
+                      confint,
+                      colors,
+                      draw_perp=False,
+                      thickness=1.8,
```

Comment:
It looks like this is being exposed here but not to the user? Is that intentional?